### DISCUSSION TITLE
sqlcmd in Backupscripts on SQL2008r2

### DISCUSSION THREAD

**Discussion by SasaJ7** [olahallengren/sql-server-maintenance-solution#568] [Questions & Answers]
Hello Ola,

Can you point me in direction, why are you still using sqlcmd in your scripts on SQL 2008r2? 
I moved to new company and here there are still few old servers, which have to be migrated to 2016 or 2019. 

Thank you!
Saša

**Comment by olahallengren** [comment]
Please see the question "Should I use CmdExec job steps with sqlcmd, or T-SQL job steps?" in the Frequently Asked Questions.
https://ola.hallengren.com/frequently-asked-questions.html
